### ISSUE TITLE
Allow all payment methods when "manual capture" is enabled

### ISSUE THREAD

**Issue by dechov** [Automattic/woocommerce-payments#5075]
Currently, payment methods that don't support deferred capture **do not appear** when "manual capture" setting is enabled (and we are communicating this to the merchant, see https://github.com/Automattic/woocommerce-payments/issues/3567). Instead, we could modify the intent to only apply the manual capture method to the specific payment methods that support it, and not affect the others.

From @jbordonado in https://github.com/Automattic/woocommerce-payments/issues/3567#issuecomment-1112276672:

<blockquote>
When capture later is enabled, we can create a payment intent such as:

- card payment method is shown at checkout and will capture later
- other payment methods are also shown at checkout and will capture automatically

We can do that by setting capture_method=automatic at the root level and override it only for the 'card' payment method in the PaymentIntent request to Stripe.

In that case, we would need a design proposition to explain the expected behaviour to the merchant whe he enables the "capture later" setting.
</blockquote>

**Comment by kalessil** [comment]
@mgascam: there are more Auth&Capture capable payment methods, can you please track the issue for the phase 2 project?

Context: p1667602754775889-slack-CGGCLBN58

**Comment by elizaan36** [comment]
> Instead, we could modify the intent to only apply the manual capture method to the specific payment methods that support it, and not affect the others.

So we'd have to inform the user that only Card payment is capture later, and all other payment methods listed are manual capture?

**Comment by RadoslavGeorgiev** [comment]
@rogermattic at Gamma, we're considering picking this issue for the next sprint. We are clear how to proceed functionality-wise, but as mentioned in the mast message by @elizaan36, we will need to change what's communicated to merchants, and maybe even consider a backwards-compatible approach so that merchants that only see some payment methods would not be surprised.

Could you please take a look?

**Comment by rogermattic** [comment]
Hey @RadoslavGeorgiev ! I'll definitely investigate that! Thanks for pinging me here.

>at Gamma, we're considering picking this issue for the next sprint.

Would that be fine if I go back to you Tuesday / Wednesday next week time-wise?

**Comment by RadoslavGeorgiev** [comment]
That would be fine @rogermattic! We don't necessarily need to start on this immediately, but we could include the issue in the next sprint.

**Comment by rogermattic** [comment]
@RadoslavGeorgiev 

I looked at the options that @jbordonado  suggested:

> **Option 1**
We just make some design changes to show that the additional payment methods won't be active (available at checkout), as it was mentioned in this comment: https://github.com/Automattic/woocommerce-payments/issues/3567#issuecomment-1007425305

That is: a modal to confirm the activation of the manual capture + greyed out other payment methods once it is enabled

and

> **Option 2**
When capture later is enabled, we can create a payment intent such as:
-card payment method is shown at checkout and will capture later
-other payment methods are also shown at checkout and will capture automatically
We can do that by setting capture_method=automatic at the root level and override it only for the 'card' payment method in the PaymentIntent request to Stripe.

And while I like [your suggestion](https://github.com/Automattic/woocommerce-payments/issues/3567#issuecomment-1115972516) to go for Option 1 as a starting point  I do agree we can iterate on it and discuss another solution.  

I also noticed that in a way Option 1 is already executed?

We do display the warning in the popup while the user wants to enable `manual capture`:

![Image](https://github.com/user-attachments/assets/dca2fb76-a25b-4989-ae20-ff5fc947b6a9)

and we also disable the payment methods that are not compatible with the `manual capture`:

![Image](https://github.com/user-attachments/assets/cbcb1d6d-d6ce-4bcd-80b7-5d6d4822bee0)

So I wonder if a third option is worth exploring:

**Option 3** (mixed)

We could approach this in the following ways:

1. Improve the notice in the modal when enabling `capture later` to ensure the message is clear and doesn’t get overlooked.
2. Then, we could:
either...
- A. Provide the user with a choice directly in the modal to either disable the non-compatible methods or enable them.
or...
- B. Add an inline notice to inform the user that some methods were disabled. Below the manual capture checkbox, include a new (initially disabled) checkbox with a label like: "Enable auto-capture for payment methods incompatible with manual capture."

In the case of option B, we would still leave the payment method checkboxes enabled, but they would include warning badges stating something like: "Manual capture disabled."

Otherwise, when the additional checkbox is disabled, the behaviour would be as in Option 1.


I wanted to run this by you before I make mockups, but let me know if that doesn't make any sense or in case you have any concerns.

**Comment by rogermattic** [comment]
Hey @RadoslavGeorgiev I noticed this got assigned to you. Let me know if you have any thoughts on my comment above, and I will proceed with the. mockups!

**Comment by RadoslavGeorgiev** [comment]
@rogermattic 

> And while I like [your suggestion](https://github.com/Automattic/woocommerce-payments/issues/3567#issuecomment-1115972516) to go for Option 1 as a starting point I do agree we can iterate on it and discuss another solution.
> I also noticed that in a way Option 1 is already executed?

The point was that Option 1 would be temporary, until we figure out if/and how to proceed with Option 2, which is what this issue is about. With that in mind, Option 1 is already in place, even if there could be some improvements to it.

As far as I can tell, in your comment, Option 2 is the same as Option 1: Disable all payment methods that do not support manual capture. What we are looking for in this issue is your Option 3.

TL;DR: This got a bit confusing, so let me paraphrase. We are looking for what you descibe as Option 3, the ability to use payment methods that do not support manual capture, even when manual capture is enabled.

---

> * A. Provide the user with a choice directly in the modal to either disable the non-compatible methods or enable them.
>   or...
> * B. Add an inline notice to inform the user that some methods were disabled. Below the manual capture checkbox, include a new (initially disabled) checkbox with a label like: "Enable auto-capture for payment methods incompatible with manual capture."
> 
> In the case of option B, we would still leave the payment method checkboxes enabled, but they would include warning badges stating something like: "Manual capture disabled."

Personally, I think that we can combine certain parts of those options:

1. Give merchants the choice in the modal (A), so everything can happen in one step.
2. Also allow them to manually select what to enable later by still allowing the checkboxes to work. It would also mekse sense to display the warning badges that you appended to option B.

Maybe the modal could look something like this:

> #### Enable manual capture
> Payments must be captured within 7 days or the authorization will expire and money will be returned to the shopper.
> Additionally, only card payments support manual capture. Your other enabled payment methods can still be available at checkout, but payments will be captured immediately. Please choose which payment methods to leave enabled:
> - [x] Giropay
> - [x] Sepa
> - [x] Method XYZ
> [Select all](#) / [Deselect all](#)
> 
> [Cancel] **[Enable Manual Capture]**  

Note: Based on Jordan's findings, Klarna and Afterpay also support manual capture, so we should keep that in mind, and alter the popup/behavior.

Does this make sense to you? If yes, please proceed with some mockups :)

**Comment by deepakpathania** [comment]
Blocked on design.

**Comment by rogermattic** [comment]
@RadoslavGeorgiev thanks for taking the time to get back to me, I know it takes a lot to talk about these options in this way :D Frankly, every time I pick up this issue it feels like quite a chunk to digest :D 

> Personally, I think that we can combine certain parts of those options:
> - Give merchants the choice in the modal (A), so everything can happen in one step.
> - Also allow them to manually select what to enable later by still allowing the checkboxes to work. It would also mekse sense to display the warning badges that you appended to option B.

Absolutely agree on the choice in modal and the badges – that's what I thought too, I'll draft something up!

> Based on Jordan's findings, Klarna and Afterpay also support manual capture, so we should keep that in mind, and alter the popup/behavior.

Does that mean they can just stay enabled as a manual capture (when the user wants to switch to manual capture) or is there any other technical complexity / nuance? Or would you say we want to give the user a choice to enable manual capture only on some payment options (out of the list that supports manual capture)?

Re: the actual message, I like your copy proposal:
>> **Enable manual capture**
>>
>>Payments must be captured within 7 days or the authorization will expire and money will be returned to the shopper.

>>Additionally, only card payments support manual capture. Your other enabled payment methods can still be available at checkout, but payments will be captured immediately. Please choose which payment methods to leave enabled:
>>
>>
>>Giropay
>>Sepa
>>Method XYZ
>>[Select all]  / [Deselect all] 
>>[Cancel] [Enable Manual Capture]

I'll just polish it a little bit while working on the mocks!

**Comment by rogermattic** [comment]
@RadoslavGeorgiev I'm just working on this issue and looking at these modals (as a starting point) I have some additional thoughts :)

I’m not sure if giving users the option to manage each method individually **still in the modal** is adding unnecessary complexity. 

At first, I considered this approach 

![Image](https://github.com/user-attachments/assets/4bff9df6-e684-4496-9172-6d5062179ccc)
where selecting `Choose manually` would allow them to pick each payment method (as you also suggested) from the list (checkboxes), but I wondered whether it might be just as easy to revise the methods later in the Payment Methods list after closing the modal, without the extra added complexity. I think it adds the complexity to the UX and to the implementation for sure too. WDYT?

I do think the options to **disable all** or **keep all as auto-capture** add value, as they save users the effort of making granular changes later and provide more context. But the ‘Choose manually’ option (or whatever we call it) feels like it might be adding complexity rather than solving a real problem. So now I'm considering this simpler modal:


<img width="598" alt="Image" src="https://github.com/user-attachments/assets/a8134f63-5da6-493b-a514-9278f0e6c8af" />

That should work nicely. WDYT? 

I'm still confident about the notice and the badges on the Payment Methods list. Will work on that, just wanted to ask you for some feedback on the above first.

**Comment by elizaan36** [comment]
I think the simplified modal makes sense @rogermattic ! Merchants then don't have to think too much, and if they would prefer some methods don't capture payments immediately they can take action to disable them in settings.

**Comment by rogermattic** [comment]
Great! Thanks Elizabeth!  

So if you have no additional feedback to it, I'll continue with the whole flow @RadoslavGeorgiev, based on this modal:

![Image](https://github.com/user-attachments/assets/d5c975f3-9b59-453a-bde4-bfc5f109fdf0)

**Comment by rogermattic** [comment]
Inline with our Slack discussion (p1744296944078989/1739443409.621449-slack-C01BPL3ALGP) the final designs (Figma: LK697bTM8sHMU95BS4owef-fi-2742_238415)  that we settled on include these changes:

- the modal design & copy tweaks,

<img width="481" alt="Image" src="https://github.com/user-attachments/assets/4e55911a-0a5b-452d-830a-53d902f55ca7" />

- adding the `Learn more` s and linking them to the [documentation](https://woocommerce.com/document/woopayments/settings-guide/authorize-and-capture/),
<img width="214" alt="Image" src="https://github.com/user-attachments/assets/5fcaac98-ad28-4e32-8b51-df6685324aaa" />

- adding the notice above the Payment methods list when manual capture is enabled (`Manual capture is enabled, so any payment methods that don't support it have been automatically disabled`)

<img width="603" alt="Image" src="https://github.com/user-attachments/assets/12c076e6-6036-4d0d-955a-4c956408191b" />
